CMakeLists.txt

```
@@ -254,6 +257,24 @@ if(GUI)
             ${Qt6OpenGLWidgets_INCLUDE_DIRS}
   )
 
+  # Dissolve QML GUI
+  qt_add_executable(${qmlgui_target_name} MACOSX_BUNDLE ${SRCS}/dissolve-gui-qml.cpp ${SRCS}/gui-qml/resources.qrc)
+
+  # Set project-local include directories for target (we expect to need less of these dependencies over time)
+  target_include_directories(
+    ${qmlgui_target_name}
+    PRIVATE ${PROJECT_SOURCE_DIR}/src
+            ${PROJECT_BINARY_DIR}/src
+            ${CMAKE_BINARY_DIR}/src/gui/gui_autogen/include
+            ${FREETYPE_INCLUDE_DIRS}
+            ${Qt6Core_INCLUDE_DIRS}
+            ${Qt6Gui_INCLUDE_DIRS}
+            ${Qt6OpenGL_INCLUDE_DIRS}
+            ${Qt6Widgets_INCLUDE_DIRS}
+            ${Qt6OpenGL_INCLUDE_DIRS}
+            ${Qt6OpenGLWidgets_INCLUDE_DIRS}
```

```suggestion
            ${Qt6Core_INCLUDE_DIRS}
            ${Qt6Gui_INCLUDE_DIRS}
```

The new GUI hopefully shouldn't need most of these libraries, and currently works without them.